community/detector/rce/cve202135464/src/test/java/com/google/tsunami/plugins/detectors/rce/cve202135464/Cve202135464DetectorTest.java

```diff
@@ -0,0 +1,157 @@
+                .setTargetInfo(targetInfo)
+                .setNetworkService(service)
+                .setDetectionTimestamp(
+                    Timestamps.fromMillis(Instant.now(fakeUtcClock).toEpochMilli()))
+                .setDetectionStatus(DetectionStatus.VULNERABILITY_VERIFIED)
+                .setVulnerability(
+                    Vulnerability.newBuilder()
+                        .setMainId(
+                            VulnerabilityId.newBuilder()
+                                .setPublisher("TSUNAMI_COMMUNITY")
+                                .setValue("CVE_2021_35464"))
+                        .setSeverity(Severity.CRITICAL)
+                        .setTitle("Pre-auth RCE in OpenAM 14.6.3/ForgeRock AM 7.0 (CVE-2021-35464)")
+                        .setDescription("OpenAM server before 14.6.3 and ForgeRock AM server before 7.0 have "
+                        + "a Java deserialization vulnerability in the jato.pageSession "
+                        + "parameter on multiple pages. The exploitation does not require "
+                        + "authentication, and remote code execution can be triggered by "
+                        + "sending a single crafted /ccversion/* request to the server. "
+                        + "The vulnerability exists due to the usage of Sun ONE Application "
+                        + "Framework (JATO) found in versions of Java 8 or earlier. The issue "
+                        + "was fixed in commit a267913b97002228c2df45f849151e9c373bc47f from "
+                        + "OpenIdentityPlatform/OpenAM:master."))
+                .build());
+  }
+
+  @Test
+  public void detect_whenFlinkNotVulnerable_returnsNoVulnerability()
+      throws IOException, InterruptedException {
+    enqueueMockVulnerabilityResponse(mockResponse);
+    NetworkService service =
+        NetworkService.newBuilder()
+            .setNetworkEndpoint(
+                forHostnameAndPort(mockWebServer.getHostName(), mockWebServer.getPort()))
+            .setTransportProtocol(TransportProtocol.TCP)
+            .setServiceName("http")
+            .build();
+    TargetInfo targetInfo =
+        TargetInfo.newBuilder()
+            .addNetworkEndpoints(forHostname(mockWebServer.getHostName()))
+            .build();
+
+    DetectionReportList detectionReports = detector.detect(targetInfo, ImmutableList.of(service));
+
+    assertThat(mockWebServer.getRequestCount()).isEqualTo(1);
+    RecordedRequest detectionReportRequest = mockWebServer.takeRequest();
+    assertThat(detectionReportRequest.getMethod()).isEqualTo("GET");
+    assertThat(detectionReportRequest.getPath()).endsWith("..;/ccversion/Version");
+    assertThat(detectionReports.getDetectionReportsList()).isEmpty();
+  }
+
+  private void enqueueMockVulnerabilityResponse() {
+    MockResponse mockResponse =
+        new MockResponse().setResponseCode(200).setPath("/..;/ccversion/Version");
+    mockWebServer.enqueue(mockResponse);
+  }
+
```

Comment:
You are missing a right curly bracket, which is causing failures in the CI pipeline.

Reply:
I fixed that but I still get some errosr, I started building using the newly added `./build_all.sh` and will check the errors in this file tomorrow.